src/components/Identity/createComponent.js

```diff
@@ -101,6 +102,12 @@ export default (
             // promise so that future requests can know when the ECID has returned.
             deferredForEcid = defer();
             payload.expectResponse();
+            if (
+              config.thirdPartyCookiesEnabled &&
+              isThirdPartyCookieSupported(window)
+            ) {
+              payload.useIdThirdPartyDomain();
+            }
```

Comment:
If I understand right, this code would call 3rd party server only when we know it supports 3rd party cookie (Chrome).

I think yesterday we were talking about call 3rd party end point on all browsers except the browsers known as blocking 3rd party (Example: Safari, Firefox and Edge).

Reply:
If you look [here](https://github.com/adobe/alloy/pull/272/files#diff-672edb84f7055556d3a2b9055ab49876R19) you'll see that "unknown" browsers are assumed to have third-party cookie support.